docs/HowTo/GetStarted/Wizard/GettingStarted.md

```diff
@@ -89,7 +89,7 @@ Clone this repo to your local machine.
 
 GoQuorum Wizard is built on open source and we invite you to contribute enhancements.
 Upon review you will be required to complete a Contributor License Agreement (CLA) before we are able to merge.
-If you have any questions about the contribution process, please feel free to send an email to [quorum@consensys.net](mailto:quorum@consensys.net).
+If you have any questions about the contribution process, please feel free to send an email to [`quorum@consensys.net`](mailto:quorum@consensys.net).
```

Comment:
Suggest pointing to the GoQuorum slack or Quorum channel on Discord rather than email.